ci-operator/config/openshift/installer/openshift-installer-master__okd-scos.yaml

```diff
@@ -1,4 +1,16 @@
 base_images:
+  ocp_4.17_base-rhel9:
```

Comment:
@sdodson do you see a problem with adding the following?

We only copy the files from those images and not use the RHEL image.
https://github.com/openshift/installer/blob/master/images/installer/Dockerfile.upi.ci

Reply:
I think we want to avoid using any RHEL images to build anything that's publicly accessible. Is this image or any image built from it made publicly accessible?

CC @Prashanth684

Reply:
iiuc, the content in the rhel base image is used by the upi-installer image which is added below. it will be part of the scos imagestream, but will not be part of the payload (which is what is publicly accessible through quay.io). This is only used by CI for installation purposes right @jcpowermac ?

Reply:
that being said, i'm adding @jupierce to this conversation as the doozer automation would probably override this configuration next time it is autogenerated, so the changes will have to come from there.

Reply:
`upi-installer` has never been apart of the release image (which seems to cause problems in general) afaik. Calling it out I just wanted to make sure that it was ok to move forward. I am not entirely clear why as is a list though

```
    ocp_4.17_base-rhel9:
      as:
      - scos-4.17
      - registry.ci.openshift.org/ocp/4.17:base-rhel9
```

and is this just replacing the from...

https://github.com/openshift/installer/blob/7a6c091248698578f5d8cdaff6b5ab27fcac742d/images/installer/Dockerfile.upi.ci#L30

Reply:
https://docs.ci.openshift.org/docs/architecture/images/#ci-images